src/SourceBuild/content/eng/build.ps1

```diff
@@ -53,26 +53,24 @@ if ($help) {
 }
 
 $project = Join-Path $RepoRoot "build.proj"
-$arguments = @()
-$targets = "/t:Build"
+$actions = @("/p:Restore=true", "/p:Build=true", "/p:Publish=true")
 
-# This repo uses the VSTest integration instead of the Arcade Test target
 if ($test) {
   $project = Join-Path (Join-Path $RepoRoot "test") "tests.proj"
-  $targets += ";VSTest"
-  $arguments += "/p:Test=true"
+  $actions = @("/p:Restore=true", "/p:Build=true", "/p:Publish=true", "/p:IsTestRun=true")
```

Comment:
Why is build.ps1 set to IsTestRun=true, why build.sh is Test=true?

Reply:
That's a bug, they should both set both properties.